lib/cylc/remote.py

```diff
@@ -59,13 +60,15 @@ def __init__(self, argv=None):
         # detect and replace host and owner options
         while argv:
             arg = argv.pop(0)
-            if arg.startswith("--user="):
-                self.owner = arg.replace("--user=", "")
-            elif arg.startswith("--host="):
-                self.host = arg.replace("--host=", "")
-            elif arg == "--login":
+            if arg.startswith('--user='):
+                self.owner = arg.replace('--user=', '')
+            elif arg.startswith('--host='):
+                self.host = arg.replace('--host=', '')
+            elif arg == '--ssh-cylc=':
```

Comment:
The ssh command generated for ssh task messaging is screwed up.  This line is the problem - it should use  `startswith()` not `==`.

Reply:
Done.